recipes/geos/all/test_v1_package/conanfile.py

```diff
@@ -13,6 +14,6 @@ def build(self):
         cmake.build()
 
     def test(self):
-        if not tools.cross_building(self):
+        if not tools.build.cross_building(self, self):
```

Comment:
```suggestion
        if not tools.build.cross_building(self):
```
We already saw this one, didn't we ?